.ci/diff-report.sh

```diff
@@ -17,27 +17,27 @@ download-files)
   curl --fail-with-body -X GET "${LINK}" \
     -H "Accept: application/vnd.github+json" \
     -H "Authorization: token $GITHUB_TOKEN" \
-    -o project.properties
+    -o ../project.properties
```

Comment:
`mkdir -p .ci-temp`

And keep all files in it. We should not go out of our repo folder.